Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -275,16 +275,26 @@ static CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localiz
     // TODO::
     // Iterate through all attributed string ranges and identify attributes so they can be used to -
     //  - set indentation
-    //  - set kern
     //  - etc.
     //  These can be done using the DWrite IDWriteTextFormat range property methods.
+
+    // Used to separate runs for attributes which DWrite does not handle until drawing (e.g. Foreground Color)
+    uint32_t incompatableAttributeFlag = 0;
+
     NSRange attributeRange;
     for (size_t i = 0; i < [ts->_attributedString length]; i += attributeRange.length) {
         NSDictionary* attribs = [ts->_attributedString attributesAtIndex:i
                                                    longestEffectiveRange:&attributeRange
                                                                  inRange:{ i, [ts->_attributedString length] - i }];
 
         const DWRITE_TEXT_RANGE dwriteRange = { attributeRange.location, attributeRange.length };
+
+        ComPtr<IDWriteTypography> typography;
```

Comment:
nit: would it be possible to put this chunk together with the chunk where it's used?